source/applications/advanced/gamma_correction.py

```diff
@@ -38,15 +36,14 @@ def adjust_gamma(image, gamma: float):
         Gamma adjusted (OpenCV) image
```

Comment:
Same here (and in the rest of the docstring). Just numpy arrays.

Reply:
done.